### DISCUSSION TITLE
Refresh on routing to specific route (/index/[id])

### DISCUSSION THREAD

**Discussion by aryan-wayne** [vercel/next.js#55175] [Help]
### Summary

Routing of specific path in Next.js make refresh the browser to the path.
Inside of [this project](https://github.com/aryan-wayne/nextjs-index-route) you can find the issue.
By running the project in home page there are 3 buttons that connected to three routes: / (as home button to home page), /asset/1 (as asset button to asset page) and /index/1 (as index button to index page).
The behavior that expected is the same behavior on all 3 buttons, but by clicking on index button the page will refresh to the path but the others have normal local routing.

### Example

https://github.com/aryan-wayne/nextjs-index-route

**Comment by msajadpourr** [comment]
i have same issue with specific path of index/[id] that make my app refresh
